src/main/java/org/prebid/server/handler/CurrencyRatesHandler.java

```diff
@@ -0,0 +1,48 @@
+package org.prebid.server.handler;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Handler;
+import io.vertx.core.json.Json;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.RoutingContext;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.prebid.server.currency.CurrencyConversionService;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+/**
+ * Handles HTTP request for currency latest update lastUpdate.
+ */
+public class CurrencyRatesHandler implements Handler<RoutingContext> {
+
+    private static final Logger logger = LoggerFactory.getLogger(CurrencyRatesHandler.class);
+
+    private final CurrencyConversionService currencyConversionService;
+
+    public CurrencyRatesHandler(CurrencyConversionService currencyConversionService) {
+        this.currencyConversionService = Objects.requireNonNull(currencyConversionService);
+    }
+
+    @Override
+    public void handle(RoutingContext context) {
+        final ZonedDateTime lastUpdated = currencyConversionService.getLastUpdated();
+        final String lastUpdatedString = lastUpdated != null ? lastUpdated.toString() : "no value";
+        try {
+            context.response().end(Json.mapper.writeValueAsString(Response.of(lastUpdatedString)));
+        } catch (IOException e) {
+            logger.error("Critical error when trying to marshal latest update response", e);
```

Comment:
Please rephrase this message to something along the lines of "Critical error when marshaling latest currency rates update response"